ui_framework/components/form/_color_picker.scss

```diff
@@ -0,0 +1,29 @@
+.kuiColorSwatch {
+  width: 19px;
+  height: 19px;
+  border-radius: 3px;
+  border: 1px solid black;
```

Comment:
Might want a softer semi-transparent color here, so it still shows a border, but isn't as harsh. Something like rgba(0,0,0,.2);